scripts/check-component-features

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Vague preference for using `/usr/bin/env bash`, but our ubuntu runners will have system bash at `/bin/bash` anyhow.